src/dmd/mtype.d

```diff
@@ -4981,18 +4985,22 @@ extern (C++) final class TypeFunction : TypeNext
         return false;
     }
 
-    /// set or get if the function has the `nothrow` attribute
-    bool isnothrow() const pure nothrow @safe @nogc
-    {
-        return (funcFlags & FunctionFlag.isnothrow) != 0;
-    }
-    /// ditto
-    void isnothrow(bool v) pure nothrow @safe @nogc
+
+    deprecated("This property was removed in favor of throw_ field")
```

Comment:
I thought that in case someone uses dmd as a library. I hear that was a thing, so I didn't remove it all to prevent breaking user code, although I don't know its state.

Reply:
Technically, it could be used, but I doubt that. I guess there's no harm in using deprecatins here.